samples/Azure.AI.DocumentTranslation/Generated/Configuration.json

```diff
@@ -7,6 +7,5 @@
     "..\\..\\..\\artifacts\\bin\\AutoRest.CSharp\\Debug\\netcoreapp3.1\\Azure.Core.Shared"
   ],
   "public-clients": true,
-  "skip-csproj-packagereference": true,
-  "data-plane": true
```

Comment:
Curious: What is the reason that `data-plane: true` is removed here?

Reply:
DPG is default target. if neither  azure-arm nor generation1-convenience-client is set, it will be DPG. No need a flag for DPG.